qds_sdk/cluster.py

```diff
@@ -499,6 +528,30 @@ def restore_point(cls, cluster_id_label, s3_location, backup_id, table_names, ov
         return conn.post(cls.element_path(cluster_id_label) + "/restore_point", data={"parameters" : parameters})
 
     @classmethod
+    def snapshot_schedule(cls, cluster_id_label, s3_location=None, frequency_unit=None, frequency_num=None, status=None):
+        """
+        Either update or get details for snapshot schedule
+        """
+        conn = Qubole.agent()
+
+        if status != None:
```

Comment:
This can be handled in the put below. Status can be changed with others. I think thats fine.